extensions/amp-app-banner/0.1/amp-app-banner.js

```diff
@@ -329,8 +330,9 @@ export class AmpAndroidAppBanner extends AbstractAppBanner {
     assertHttpsUrl(this.manifestHref_, this.element, 'manifest href');
 
     /** @private @const {!Element} */
-    this.openLink_ = user().assert(this.element.querySelector('a[open-link]'),
-        '<a open-link> is required inside %s: %s', TAG, this.element);
+    this.openButton_ = user().assert(
+        this.element.querySelector('button[open-link]'),
```

Comment:
`test-example-validation.js` has a temporary rule for `/DISALLOWED_ATTR open-link a/` probably needs to be updated to be `button` instead of `a`